src/nodedef.cpp

```diff
@@ -676,12 +676,12 @@ void ContentFeatures::updateTextures(ITextureSource *tsrc, IShaderSource *shdsrc
 
 	bool is_liquid = false;
 
-	u8 material_type = (alpha == 255) ?
-		TILE_MATERIAL_BASIC : TILE_MATERIAL_ALPHA;
+	u8 material_type = TILE_MATERIAL_ALPHA;
 
 	switch (drawtype) {
 	default:
 	case NDT_NORMAL:
+		material_type = TILE_MATERIAL_BASIC;
 		solidness = 2;
 		break;
 	case NDT_AIRLIKE:
```

Comment:
Hmm well nodeboxes are automatically converted into meshes now. Nodeboxes do support transparency, for example you can have glass slabs.